shell/detail/provisioning.cattle.io.cluster.vue

```diff
@@ -101,93 +78,12 @@ export default {
-    }
-
-    if ( this.$store.getters['management/canList'](CAPI.MACHINE) ) {
-      fetchOne.machines = this.$store.dispatch('management/findAll', { type: CAPI.MACHINE });
-    }
-
-    if ( this.$store.getters['management/canList'](SNAPSHOT) ) {
-      fetchOne.snapshots = this.$store.dispatch('management/findAll', { type: SNAPSHOT });
-    }
-
     if ( this.value.isImported || this.value.isCustom || this.value.isHostedKubernetesProvider ) {
-      fetchOne.clusterToken = this.value.getOrCreateToken();
-    }
-
-    // Need to get Norman clusters so that we can check if user has permissions to access the local cluster
-    if ( this.$store.getters['rancher/canList'](NORMAN.CLUSTER) ) {
-      fetchOne.normanClusters = this.$store.dispatch('rancher/findAll', { type: NORMAN.CLUSTER });
-    }
-
-    if ( this.value.isRke1 && this.$store.getters['isRancher'] ) {
-      fetchOne.etcdBackups = this.$store.dispatch('rancher/findAll', { type: NORMAN.ETCD_BACKUP });
-
-      fetchOne.normanNodePools = this.$store.dispatch('rancher/findAll', { type: NORMAN.NODE_POOL });
-    }
-
-    const fetchOneRes = await allHash(fetchOne);
-
-    this.allMachines = fetchOneRes.machines || [];
-    this.allMachineDeployments = fetchOneRes.machineDeployments || [];
-    this.haveMachines = !!fetchOneRes.machines;
-    this.haveDeployments = !!fetchOneRes.machineDeployments;
-    this.clusterToken = fetchOneRes.clusterToken;
-    this.etcdBackups = fetchOneRes.etcdBackups;
-
-    if (fetchOneRes.normanClusters) {
-      // Does the user have access to the local cluster? Need to in order to be able to show the 'Related Resources' tab
-      this.hasLocalAccess = !!fetchOneRes.normanClusters.find((c) => c.internal);
-    }
-
-    const fetchTwo = {};
-
-    const thisClusterMachines = this.allMachineDeployments.filter((deployment) => {
-      return deployment?.spec?.clusterName === this.value.metadata.name;
-    });
-
-    const machineDeploymentTemplateType = thisClusterMachines?.[0]?.templateType;
-
-    if (machineDeploymentTemplateType && this.$store.getters['management/schemaFor'](machineDeploymentTemplateType) ) {
-      fetchTwo.mdtt = this.$store.dispatch('management/findAll', { type: machineDeploymentTemplateType });
-    }
-
-    if (!this.showMachines) {
-      if ( this.$store.getters['management/canList'](MANAGEMENT.NODE) ) {
-        fetchTwo.allNodes = this.$store.dispatch('management/findAll', { type: MANAGEMENT.NODE });
-      }
-
-      if ( this.$store.getters['management/canList'](MANAGEMENT.NODE_POOL) ) {
-        fetchTwo.allNodePools = this.$store.dispatch('management/findAll', { type: MANAGEMENT.NODE_POOL });
-      }
-
-      if ( this.$store.getters['management/canList'](MANAGEMENT.NODE_TEMPLATE) ) {
-        fetchTwo.nodeTemplates = this.$store.dispatch('management/findAll', { type: MANAGEMENT.NODE_TEMPLATE });
-      }
+      this.clusterToken = await this.value.getOrCreateToken();
```

Comment:
I can prolly await on both this and the hasLocalCluster to speed things up slightly.